### ISSUE TITLE
DBeaver 6.1.5+ not loading extra data-sources-*.json configuration files

### ISSUE THREAD

**Issue by MannerMan** [dbeaver/dbeaver#6700]
#### System information: 
- Operating system: Ubuntu 18.04
- DBeaver version: 6.2.0
- Additional extensions: None

#### Connection specification:
- Database name and version: postgres 9.5 & 10 ( probably not relevant )

#### Describe the problem you're observing:
Dbeaver documentation describes the possibility to define several connection data sources, by having multiple files named according to the pattern data-sources-*.json. ( https://github.com/dbeaver/dbeaver/wiki/Admin-Manage-Connections ) 

I have used this functionality with the xml -style configuration to generate additional configuration files with connection config for various servers. I could name these extra files according to the data-sources-*.xml pattern and dbeaver would merge all configuration.

It appears this is not working with the new json-format. Adding a `data-sources-2.json` file in `/home/$user/.local/share/DBeaverData/workspace6/General/.dbeaver/` has no effect, its configuration is not loaded by DBeaver. Overwriting `data-sources.json` with the contents of `data-sources-2.json` does make the configuration appear, at the loss of the manually defined connections. 

I have tested both DBeaver 6.1.5 and 6.2 with the same result.

#### Steps to reproduce, if exist:
Create a `data-sources-2.json` file in `/home/$user/.local/share/DBeaverData/workspace6/General/.dbeaver/` with the contents below and restart DBeaver:
```
{
  "folders": {},
  "connections": {
    "postgres-jdbc-16d15bc75ef-1016ec509473e93a": {
      "provider": "postgresql",
      "driver": "postgres-jdbc",
      "name": "PostgreSQL - postgres 2",
      "save-password": true,
      "read-only": false,
      "configuration": {
        "host": "localhost",
        "port": "5432",
        "database": "postgres",
        "url": "jdbc:postgresql://localhost:5432/postgres",
        "type": "dev",
        "provider-properties": {
          "@dbeaver-show-non-default-db@": "false",
          "@dbeaver-show-template-db@": "false"
        },
        "handlers": {}
      }
    }
  }
}
```
_Expected result:_ A connection named "PostgreSQL - postgres 2" should appear.

_Actual result:_ No new connection is listed.

#### Include any warning/errors/backtraces from the logs
No errors are logged as far as I can see, neither in dbeaver-debug.log or in stdout when launching dbeaver from terminal.

**Comment by serge-rider** [comment]
Yes. DBeaver doesn't refresh changed resources during startup. Thanks for the bug report.

Workaround: refresh project to see config changes (select your project in "Projects" view and press F5).

**Comment by MannerMan** [comment]
@serge-rider Thanks, your workaround did the trick!

**Comment by uslss** [comment]
verified

**Comment by claudio-silva** [comment]
I'm sorry to comment on this old issue, but I'm still facing the same problem.
Pressing F5 solved it, BUT had I not searched the Internet for a long time untill I found this GitHub issue, I would have had no idea of how to solve it. And this will surely happen to other users.
So DBeaver is still not refreshing the project on startup from the json config files.
In my case, I copied the config. files from a Time Machine backup, on a Mac, and DBeaver would not show the connections and bookmarks.
